### PR TITLE
Add Minds as easy

### PR DESCRIPTION
**Pull request by 1t3x**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -7274,7 +7274,14 @@
             "mindmeister.com"
         ]
     },
-
+    {
```
Comment (tupaschoal) [inline _data/sites.json:7277]:
```suggestion

    {
```

File: _data/sites.json [hunk 2]
```diff
@@ -7274,7 +7274,14 @@
...
+        "name": "Minds",
+        "url": "https://www.minds.com/settings/other/delete-account",
+        "difficulty": "easy",
+        "domains": [
+            "minds.com"
+        ]
+    },
```
Comment (tupaschoal) [inline _data/sites.json:7284]:
```suggestion
    },

```

File: _data/sites.json [hunk 3]
```diff
@@ -7275,6 +7275,14 @@
...
     },
 
+    {
+        "name": "Minds",
+        "url": "https://www.minds.com/settings/other/delete-account",
+        "difficulty": "easy",
+        "domains": [
+            "minds.com"
+        ]
+     },
```
Comment (tupaschoal) [inline _data/sites.json:7285]:
```suggestion
    },

```

File: _data/sites.json [hunk 4]
```diff
@@ -7275,6 +7275,14 @@
...
     },
 
+    {
+        "name": "Minds",
+        "url": "https://www.minds.com/settings/other/delete-account",
+        "difficulty": "easy",
+        "domains": [
+            "minds.com"
+        ]
+    },
```
Comment (tupaschoal) [inline _data/sites.json:7285]: Add an empty line after this one please, I'm trying to, but Web interface is not allowing

**Comment by 1t3x** [comment]
Here, I added the line

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**